src/index.html

```diff
@@ -4,14 +4,166 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalabel=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="form-group user-info_group"> 
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            minlength="2"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input 
+            type="text"
+            name="name"
+            minlength="2"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birth"
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+            required
+          >
+        </label>
+      </fieldset>
+    
+      <fieldset class="form-group registration_group">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email" 
+            name="e-mail" 
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="8" 
+            maxlength="32" 
+            required
+          >
+        </label>       
+      </fieldset>
+
+      <fieldset class="form-group user-facts_group">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?         
+          <input 
+            type="radio"
+            name="loveCat"
+            value="yes"
+          >
+          Yes
+          <input 
+            type="radio" 
+            name="loveCat" 
+            value="no"
+          >
+          No
+        </label>
```

Comment:
add separate labels for inputs. when I click on `Do you love cats?` it selects the first option now (`Yes`).